src/app/containers/ArticleTimestamp/timeFormatTests/expectedFormats.json

```diff
@@ -101,10 +101,10 @@
   },
   "gujarati": {
     "default": {
-      "one minute ago": "એક મિનિટ પહેલા",
-      "five minutes ago": "5 મિનિટ પહેલા",
-      "one hour ago": "એક કલાક પહેલા",
-      "five hours ago": "5 કલાક પહેલા",
```

Comment:
Looks like a slight difference in formatting here. May not be an issue as it seems to only affect this service.

Reply:
Can keep an eye on it! However, there's a PR and ticket in the body description to address the downgrading in this PR of Moment (which may be the culprit) and to re-upgrade it post-deployment which might solve any formatting discrepancies.